tests/Event/samples/event_with_dummy_location.cdbxml.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<cdb:event xmlns:cdb="http://www.cultuurdatabank.com/XMLSchema/CdbXSD/3.2/FINAL" availablefrom="2014-11-05T00:00:00" availableto="2014-11-22T00:00:00" cdbid="dcd1ef37-0608-4824-afe3-99124feda64b" createdby="gentonfiles@gmail.com" creationdate="2014-11-05T13:23:36" externalid="CDB:a0a2120e-34f9-4653-a504-09550475bc84" isparent="false" lastupdated="2014-11-08T15:36:06" lastupdatedby="sdebuck1" pctcomplete="85" published="true" owner="Invoerders Algemeen " private="false" validator="Gent Validatoren" wfstatus="approved">
+    <cdb:activities>
+        <cdb:activity count="0" type="like"/>
+        <cdb:activity count="0" type="attend"/>
+        <cdb:activity count="0" type="comment"/>
+        <cdb:activity count="0" type="recommend"/>
+        <cdb:activity count="0" type="facebook_share"/>
+        <cdb:activity count="0" type="review"/>
+    </cdb:activities>
+    <cdb:calendar>
+        <cdb:timestamps>
+            <cdb:timestamp>
+                <cdb:date>2014-11-21</cdb:date>
+                <cdb:timestart>20:00:00</cdb:timestart>
+            </cdb:timestamp>
+        </cdb:timestamps>
+    </cdb:calendar>
+    <cdb:categories>
+        <cdb:category catid="reg.359" type="flanderstouristregion">Kunststad Gent</cdb:category>
+        <cdb:category catid="0.50.21.0.0" type="eventtype">Spel of quiz</cdb:category>
+        <cdb:category catid="reg.1258" type="flandersregion">9000 Gent</cdb:category>
+    </cdb:categories>
+    <cdb:comments/>
+    <cdb:eventdetails>
+        <cdb:eventdetail lang="nl">
+            <cdb:calendarsummary>vrij 21/11/14 om 20:00 </cdb:calendarsummary>
+            <cdb:shortdescription>Korte beschrijving</cdb:shortdescription>
+            <cdb:title>'Bekend met Gent' - quiz</cdb:title>
+        </cdb:eventdetail>
+    </cdb:eventdetails>
+    <cdb:keywords>gent;Quiz;;Gent on Files</cdb:keywords>
```

Comment:
Are the 2 `;;` intentional?

Reply:
They are not invalid according to the `XSD`, but it might not yield the expected results, I assume they were copied from other tests.

Reply:
It was indeed a copy from other tests. I will just remove the keywords.